F1/F1/Race/RaceViewModel.swift

```diff
@@ -9,45 +9,65 @@ import Foundation
 
 class RaceViewModel {
 
+    // MARK: Variables
     private var repository: RaceRepositoryType?
     private weak var delegate: ViewModelDelegate?
     private(set) var race: RaceInfo?
-    private(set) var sortedRaceSessions: [(key: String, value: RaceSessions)] = []
+    private(set) var sortedRaceSession: [RaceSessionDetail] = []
 
     init(repository: RaceRepositoryType,
          delegate: ViewModelDelegate) {
         self.repository = repository
         self.delegate = delegate
     }
 
-    func raceSessionsCalculator() {
-        var raceSessions: [String: RaceSessions] = [:]
-        raceSessions["Race"] = RaceSessions(date: race?.date ?? "", time: race?.time ?? "")
-        raceSessions["Practice 1"] = race?.firstPractice
-        raceSessions["Qaulifying"] = race?.qualifying
+    // MARK: Computed
+    var scheduleCount: Int {
+        sortedRaceSession.count
+    }
+
+    var raceTitle: String {
+        race?.circuit.location.country ?? ""
+    }
+
+    var raceName: String {
+        race?.raceName ?? ""
+    }
+
+    var raceLocation: String {
+        "\(race?.circuit.location.locality ?? "") | \(race?.circuit.location.country ?? "")"
+    }
+
+    // MARK: Functions
+    func processRaceSessions() {
+
```

Comment:
can remove this new line for consistency. I see you don't have a new line in the computed variables, so keep to that structure with the functions as well then